shub/login.py

```diff
@@ -1,21 +1,56 @@
-import os, re, click
-from shub.utils import get_key_netrc, NETRC_FILE
+import re
+import click
+import os
+from shub import auth
+from scrapy.utils.conf import get_config
```

Comment:
I considered copying scrapy.utils.conf when I noticed all the deps that were being loaded just for it. On the other hand, does it make sense to use shub if the user is not working on a scrapy project?